docs/swagger.yaml

```
@@ -23,7 +23,7 @@ paths:
               schema:
                 $ref: '#/components/schemas/TableResponse'
   /tables/{tableId}:
-    post:
+    get:
       tags:
         - table
```

@wataaki 
僕が作ったところで申し訳ないけど、タグは複数形がよいかも！
```suggestion
        - tables
```